src/sentinelio.jl

```diff
@@ -270,7 +270,10 @@ function Base.seek(s::SentinelIO, n::Integer)
     return s
 end
 
-Base.seekend(s::SentinelIO) = read(s) # read until the end
+function Base.seekend(s::SentinelIO)
+    read(s) # read until the end
```

Comment:
After playing around with this for a bit, I think a marginally better way to perform seeking on SentinelIO streams is to read into a null stream. That way you do not need to allocate an array to hold the result of the read. It's especially useful if you need to seek a long way until you find the EOF sentinel.
```suggestion
    write(devnull, s) # read until the end
```